### ISSUE TITLE
Disable shoot animation if out of bullets.

### ISSUE THREAD

**Issue by ghost** [eloipuertas/ES2013Dimarts#249]
If player is out of bullets gun keeps "shooting", disable animation for better user feedback.

**Comment by ghost** [comment]
Tambien pasa cuando estas en menu pause, sigue disparando el fogonazo pero no la animación

**Comment by ghost** [comment]
(El eloy lo ha comentado ya e otro issueby podrian unirse estos dos)

**Comment by hiroCat** [comment]
already open #243

**Comment by ghost** [comment]
Cierto, fallo mío. Me tengo que mirar mejor las issues abiertas, es que al ser "externo" no sé como tenéis el tema.

**Comment by hiroCat** [comment]
don't worry ! and welcome !

**Comment by eloipuertas** [comment]
Cap problema si en Beta-Testing es reporta algun bug que ja està sent-se tractat. El descartarem i ja està XD. La gràcia del beta-tester és que sigui totalment extern! Gràcies @emanzanomarcos per la teva ajuda altruista!!!!

**Comment by dcorto** [comment]
La marcamos la issue con la label "invalid" por estar duplicada y así evitamos confusiones futuras :)
